config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -120,6 +120,45 @@ periodics:
     testgrid-alert-email: skclark@redhat.com
     description: "Uses kubetest to run serial node-e2e tests (+Serial, -Flaky|Benchmark|Node*Feature)"
 
+- name: ci-kubernetes-node-arm64-ubuntu-serial
+  interval: 4h
+  cluster: k8s-infra-prow-build
+  annotations:
+    testgrid-dashboards: sig-node-kubelet
+    testgrid-tab-name: kubelet-gce-e2e-arm64-ubuntu-serial
+    description: "Run serial node e2e tests on ARM64 environment on Ubuntu"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230207-192d5afee3-master
+        args:
+          - --repo=k8s.io/kubernetes=master
+          - --timeout=240
+          - --root=/go/src
+          - --scenario=kubernetes_e2e
+          - --
+          - --deployment=node
+          - --env=KUBE_BUILD_FOR_ARM64=true
```

Comment:
@ike-ma where is this used? `KUBE_BUILD_FOR_ARM64`

Reply:
I was using it to control whether to build for ARM64 or AMD64: https://github.com/kubernetes/kubernetes/pull/117017/files/c7c8c7fe388079adf3a24e0d465dd35afea15357#diff-4e140fc3b0b901a6cdad3f45948687ebd709753a4c6b48a52e876a1fec856247

Reply:
Is this still relevant? It seems to be removed from kubernetes/kubernetes#117017

Reply:
+1, this should no longer be needed?

Reply:
This has been removed